astroquery/ipac/irsa/most/core.py

```diff
@@ -0,0 +1,1017 @@
+        inclination : float or None
+            Inclination (0-180 degrees).
+            Required when input mode is ``"manual_input"``.
+        arg_perihelion : str or None
+            Argument of perihelion (0-360 degrees).
+            Required when input mode is ``"manual_input"``.
+        ascend_node : float or None
+            Longitude of the ascending node (0-360).
+            Required when input mode is ``"manual_input"``.
+        semimajor_axis : float or None
+            Semimajor axis (AU).
+            Required when input mode is ``"manual_input"`` and object type is
+            ``"Asteroid"``.
+        mean_anomaly : str or None
+            Mean anomaly (degrees).
+            Required when input mode is ``"manual_input"`` and object type is
+            ``"Asteroid"``.
+        perih_dist : float or None
+            Perihelion distance (AU).
+            Required when input mode is ``"manual_input"`` and object type is
+            ``"Comet"``.
+        perih_time : str or None
+            Perihelion time (YYYY+MM+DD+HH:MM:SS).
+            Required when input mode is ``"manual_input"`` and object type is
+            ``"Comet"``.
+        get_query_payload : bool
+            Return the query parameters as a dictionary. Useful for debugging.
+            Optional.
+            Default: ``False``
+        save : bool
+            Whether to save the file to a local directory.
+        savedir : str
+            The location to save the local file if you want to save it
+            somewhere other than `~astroquery.query.BaseQuery.cache_location`
+
+        Returns
+        -------
+        images : list
+            A list of `~astropy.io.fits.HDUList` objects.
+        """
+        # We insist on output_mode being regular so that it executes quicker,
+        # and we insist on tarballs so the download is quicker. We ignore
+        # whatever else user provides, but leave the parameters as arguments to
+        # keep the same signatures for doc purposes.
+        queryres = self.query_object(
+            catalog=catalog,
+            input_mode=input_mode,
+            obs_begin=obs_begin,
+            obs_end=obs_end,
+            ephem_step=ephem_step,
+            obj_name=obj_name,
+            obj_nafid=obj_nafid,
+            obj_type=obj_type,
+            mpc_data=mpc_data,
+            body_designation=body_designation,
+            epoch=epoch,
+            eccentricity=eccentricity,
+            inclination=inclination,
+            arg_perihelion=arg_perihelion,
+            ascend_node=ascend_node,
+            semimajor_axis=semimajor_axis,
+            mean_anomaly=mean_anomaly,
+            perih_dist=perih_dist,
+            perih_time=perih_time,
+            get_query_payload=get_query_payload,
+            output_mode="Regular",
+            with_tarballs=True,
+        )
+
+        request = self._request("GET", queryres["fits_tarball"],
```

Comment:
super nitpick as this level of consistency doesn't matter, but I believe we tend to call this `response` instead of `request`.